bundles/framework/layerlist/resources/scss/style.scss

```diff
@@ -1,5 +1,21 @@
 $filter-border-color: #a4a4a4;
 
+@keyframes blinking {
+    0% {opacity: 1;}
+    50% {opacity: 0.25;}
+    100% {opacity: 1;}
+}
+
+div.blink {
```

Comment:
This can very easily bleed outside of the intended element. Maybe move it a couple of lines lower under the div.layerselector2 definition to scope it more tightly.

Reply:
We don't have dom elements with class `layerselector2`. It should be removed.

Reply:
Right :) So maybe this should be defined as a styled-component?